lib/private/Config/Lexicon/CoreConfigLexicon.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
```

Comment:
```suggestion
 * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
```
Happy new year!

But nitpicking really. :door: :runner: